weblate/trans/forms.py

```diff
@@ -197,80 +197,89 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def get_rtl_toolbar(self, fieldname):
-        groups = []
-
         # Special chars
-        chars = []
-        for name, char, value in RTL_CHARS_DATA:
-            chars.append(
-                BUTTON_TEMPLATE.format(
-                    "specialchar",
-                    name,
-                    'data-value="{}"'.format(
-                        value.encode("ascii", "xmlcharrefreplace").decode("ascii")
-                    ),
-                    char,
-                )
-            )
+        chars = format_html_join(
+            "\n",
+            BUTTON_TEMPLATE,
+            (
+                ("specialchar", name, format_html('data-value="{}"', value), char)
```

Comment:
I think `encode("ascii", "xmlcharrefreplace").decode("ascii")` was used here to make sure that RTL direction markers do not confuse browsers (maybe it was an issue when looking at the page source, but I don't recall details).

Reply:
Not the biggest RTL expert, but looks like nothing broke. Having the page look like this, it produces the same HTML output (I've checked with `diff`):
![image](https://user-images.githubusercontent.com/39462442/170126623-6e26d8d0-0e3b-48f2-b1a9-e6db081138e8.png)

Reply:
Whoops. It's because I copied it from devtools. Tried with `curl`, and indeed there is difference. Will search for a way.